core/templates/global_assets/scripts/espresso_core.js

```diff
@@ -399,12 +399,14 @@ jQuery( document ).ready( function ( $ ) {
                         .hide()
                         .children('option')
                         .prop('selected', false)
-                        .prop('disabled', true);
+                        .prop('disabled', true)
+                        .hide();
```

Comment:
The first .hide() runs on the parent (the optgroup element), then we pull all of its .children() (the select elements) and make sure they aren't 'selected', disabled them and now hide them as well (the second .hide() ).

So with just the hide at the end, you'll end up leaving the optgroup label in the dropdown, meaning it will look something like this:

https://monosnap.com/file/7NyrRC1yOQW3Nt7xIRpt0LundpXM0B

> ...don't even remember my own code 🤦🏻‍♂️

LOL, wouldn't expect you to tbh, this was added wayyy back and EE ain't exactly small :)

Reply:
kk thnx... APPROVED 💯